doc/whats_new/v1.0.rst

```diff
@@ -192,6 +192,9 @@ Changelog
   polynomial ``degree`` of the splines, number of knots ``n_knots`` and knot
   positioning strategy ``knots``.
   :pr:`18368` by :user:`Christian Lorentzen <lorentzenchr>`.
+- |Enhancement| The :class:`preprocessing.SplineTransformer` supports periodic
+  splines via the ``extrapolation`` argument.
+  :pr:`19483` by :user:`Malte Londschien <mlondschien>`.
```

Comment:
Since `SplineTransformer` is introduced in the same release, marking this as enhancement is not really correct. I would just collapse this into the previous entry.

```suggestion
  :pr:`18368` by :user:`Christian Lorentzen <lorentzenchr>`.
  :class:`preprocessing.SplineTransformer` also supports periodic
  splines via the ``extrapolation`` argument.
  :pr:`19483` by :user:`Malte Londschien <mlondschien>`.
```